### ISSUE TITLE
Add new account - autofill ime option for password managers

### ISSUE THREAD

**Issue by Djfe** [thunderbird/thunderbird-android#7404]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.712

### Problem you are trying to solve

I would be nice if K-9s new account creation screen would support Android's newish ime autofill option.
allowing me to fill the inputs with data from my password manager.
Currently I have to paste passwords manually since my keyboard doesn't display the autofill option at all, neither for mail nor password fields.

### Suggested solution

[what is ime autofill?](https://developer.android.com/guide/topics/text/ime-autofill)
[how it looks like on the keyboard app](https://www.xda-developers.com/swiftkey-keyboard-suggestions-easier-enter-passwords-send-replies/)

**Comment by Djfe** [comment]
This is similar to #6244 but not the same. I am exclusively asking for IME autofill support while the other ticket asked for both inputs to be fillable at once (with the old-style password manager overlay, not the new one)

**Comment by Djfe** [comment]
Ok you might say it's a duplicate of https://github.com/thunderbird/thunderbird-android/issues/3646#issuecomment-429172116
But that issue is 5 years old and some things might have changed since then/I wanted to bring up IME autofill.

**Comment by wmontwe** [comment]
The IME autofill currently doesn't work very well with the Jetpack Compose UI. We wan't to upgrade from Material 2 to 3 first and then reevaluate how this could be supported.

**Comment by Djfe** [comment]
I just verified this is neither working for enpass nor lastpass.
Android 14, Pixel 7 Pro

**Comment by kv1dr** [comment]
I checked this bug and here are my findings:
With current stable version of androidx.autofill:autofill (1.1.0) it's really hard to implement autofill, mostly because documentation is missing, or at least I had hard time finding it. Android documentation is talking about latest release candidate version, that is not stable yet. So I tried implementing with latest rc version (1.3.0-rc01) and I was successful, with only a line of change or two. But the problem is, that we need to also update androidx.compose.ui:ui to rc (1.8.0-rc2, current stable version is 1.7.8), so now we have two packages that are not stable yet. I would assume they will become stable sooner or later, since they are rc, but until they are stable, I wouldn't merge this into main. Changes I made are on this branch: https://github.com/kv1dr/thunderbird-android/commits/fix-issue-7404/. Fill free to check it out.

**Comment by kewisch** [comment]
This is great news! Can you keep an eye on the releases and send a PR when done? You could also send a draft PR and we'll mark it blocked, though if you expect it will be a while until the rc becomes final then we could keep it here. I just want to make sure your effort doesn't get lost. Thank you :)

**Comment by kv1dr** [comment]
Yeah, sure, will do. I can make PR and mark it as draft. I think compose.ui could be stable soon, since latest alpha was in January, 3 beta releases were in February, and two rc releases were in March, so they are pretty quick with their releases. But autofill is worrying me a little bit. Alpha was released in May 2023, beta in September 2024, and now rc in February 2025, although there were no change between beta and rc. So it may take a while. I also found a ticket with "Graduate autofill 1.3.0 to stable" title on google issue tracker, so there is still hope. I also turn on notifications on that ticket. Will still make a draft PR, so we don't lose changes.

**Comment by kv1dr** [comment]
FYI, androidx.compose.ui 1.8.0 is now stable, while autofill is still on 1.3.0-rc01.
